lib/plain/endpoints/environment.ts

```diff
@@ -13,6 +14,10 @@ export const get = (http: AxiosInstance, params: GetEnvironmentParams) => {
   )
 }
 
+export const getAll = (http: AxiosInstance, params: GetSpaceParams) => {
```

Comment:
That endpoint is paginated, so it should be getMany with additional query params.

Reply:
it's not paginated according to the documentation and the current implementation of the main client.

Reply:
Interestingly, the API does answer with a paginated request and I am able to utilise skip and limit.

Reply:
pricing limits are always below 100 envs, so all our API clients expose `getAll` instead of `getMany`